packages/components/src/particles/ComboBox/ComboBox.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { ComponentProps, For, createMemo, createSignal } from 'solid-js';
+import ComboBox, { ComboBoxProps } from '.';
+import { randomNames } from '../../sampleData/randomNames';
+
+type InputGroupStory = StoryObj<ComboBoxProps>;
+
+const meta: Meta<ComponentProps<typeof ComboBox>> = {
+  title: 'ComboBox',
+  // @ts-ignore
+  component: ComboBox,
+  tags: ['autodocs'],
+  argTypes: {}
+};
+
+export default meta;
+
+export const Default: InputGroupStory = {
+  // @ts-ignore
+  render: (args) => {
+    const [query, setQuery] = createSignal('');
+    const filteredPeople = createMemo(() => {
+      return query() === ''
+        ? randomNames
+        : randomNames.filter((person) => {
+            return person.name.toLowerCase().includes(query().toLowerCase());
+          });
+    });
+
+    return (
+      <ComboBox {...args}>
+        <ComboBox.Input onInput={(e) => setQuery((e.target as HTMLInputElement).value)} />
+        <ComboBox.Options>
+          <For each={filteredPeople()}>
+            {(person) => (
+              <ComboBox.Option key={person.id} value={person.name}>
+                {person.name}
```

Comment:
Do we need value and child value?

Reply:
I think this interface is going to change, specifically what values look like, but the child value might be different for example how it's happening inside of `PharmacySearch`:

``` jsx
<ComboBox.Option
  key={pharmacy.id}
  value={`${pharmacy.name}, ${formattedAddress}`}
>
  <div>{pharmacy.name}</div>
  <div class="text-xs">{formattedAddress}</div>
</ComboBox.Option>
```